model/src/test/kotlin/PackageCurationTest.kt

```diff
@@ -409,6 +410,22 @@ class PackageCurationTest : WordSpec({
     }
 
     "isApplicable()" should {
+        "accept empty an name and / or version" {
+            val curation = PackageCuration(
+                id = Identifier("Maven:com.android.tools"),
+                data = PackageCurationData(
+                    authors = sortedSetOf("The Android Open Source Project"),
+                    concludedLicense = "Apache-2.0".toSpdx()
+                )
+            )
+
+            assertSoftly {
+                curation.isApplicable(Identifier("Maven:com.android.tools:common:25.3.0")) shouldBe true
+                curation.isApplicable(Identifier("Maven:com.android.tools:common")) shouldBe true
+                curation.isApplicable(Identifier("Maven:com.android.tools")) shouldBe true
```

Comment:
Add `curation.isApplicable(Identifier("Maven:com.android.tools::25.2.0")) shouldBe true` to test all cases (IIRC).